### ISSUE TITLE
[Audio Player]: QA: Text transcript is not available for audio content

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9287]
### Description

Page Element: Audio content

Issue Description:
Text transcript is not available for audio content

Expected:
Text transcript should be available for audio content

User Impact:
When no text transcript is available, users who are deaf will not be able understand the audio content completely.

### Component(s) impacted

Audio player

### Browser

Firefox

### Carbon for IBM.com version

v1.37.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/react-experimental/?path=/story/components-audio-player--default

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by oliviaflory** [comment]
Suggestion: remove closed captions and replace with Download transcript.
![Screen Shot 2022-08-29 at 11 10 28 AM](https://user-images.githubusercontent.com/45692622/187234080-bc0853f5-ae21-42b2-a6be-0538f862a10b.png)

**Comment by tweenn** [comment]
Hey @gopinathan-n and @oliviaflory - according to Able@IBM [requirements list](https://www.ibm.com/able/requirements/requirements), on item 1.2.2 the following is present:

> The source material for captions can also serve as a transcript or be transformed by assistive technology into other formats.

If the caption is available and played alongside the video, is there a need to have the file downloaded into the user machine?
I'm asking this 'cause there may be a legal issue around downloading the file.
